onnxruntime/core/providers/cpu/ml/ml_common.h

```diff
@@ -324,16 +325,20 @@ static void write_scores(std::vector<IT>& scores, POST_EVAL_TRANSFORM post_trans
         for (auto it = scores.cbegin(); it != scores.cend(); ++it, ++Z)
           *Z = static_cast<T>(ComputeLogistic(static_cast<float>(*it)));
         break;
-      case POST_EVAL_TRANSFORM::SOFTMAX:
-        ComputeSoftmax(scores);
+      case POST_EVAL_TRANSFORM::SOFTMAX: {
+        auto span = gsl::make_span(scores);
```

Comment:
>gsl::make_span(scores); [](http://example.com/codeflow?start=20&length=23)

You can pass containers into functions that accept span, will automatically covert.

Reply:
I think `ComputeSoftmax(scores)` may work if ComputeSoftmax is updated to take a `gsl::span<T>` or `const gsl::span<T>&` parameter

Reply:
See next comment.

Reply:
got it. the argument needs to be `span<const T>` to work